development/prow-installer/before-commit.sh

```diff
@@ -14,13 +16,15 @@ echo -e "${INVERTED}"
 echo "USER: " + "$USER"
 echo "PATH: " + "$PATH"
 echo "GOPATH:" + "$GOPATH"
+echo "CURRENT DIRECTORY: $DIR"
```

Comment:
I assume you used this for debugging, do we still need this? extra line?

Reply:
I think we can leave it as it is. Having current directory shown up should ease future debugging.
I might delete it as well

Reply:
The above line is just to ensure that current gopath/bin is set in path as a workaround.